src/Spryker/Zed/EventBehavior/Business/EventBehaviorFacadeInterface.php

```diff
@@ -112,6 +114,19 @@ public function getEventTransfersOriginalValues(array $eventTransfers, string $c
      */
     public function getEventTransfersAdditionalValues(array $eventTransfers, string $columnName): array;
 
+    /**
+     * Specification:
+     *    - Returns fields with timestamp in eventTransfers.
+     *    - Structure: HydrateEventsResponseTransfer.*TimestampMap Key: id, Value: timestamp.
+     *
```

Comment:
Please add a full specification for this. What is input, what are the options to use and when, how the options influence the response, etc?

Be as clear as possible; no one will understand it otherwise.

> Returns fields with timestamp in eventTransfers.
Is not true. You return a HydrateEventsResponseTransfer with different values being set based on the input you give.